_includes/sidebar.html

```diff
@@ -43,5 +43,6 @@
     </p>
 
     <p>&copy; {{ site.time | date: '%Y' }} <a href="https://github.com/doomspork">Sean Callan</a></p>
+    <p><a href="/sitemap.xml">sitemap</a></p>
```

Comment:
Once It's a XML instead an html, is that required? My previous projects they used to have a html and xml version of sitemap.